airbyte-commons-server/src/main/java/io/airbyte/commons/server/handlers/WebBackendCheckUpdatesHandler.java

```diff
@@ -32,11 +32,11 @@ public class WebBackendCheckUpdatesHandler {
   // todo (cgardens) - this handler should NOT have access to the db. only access via handler.
   @Deprecated
   final ConfigRepository configRepositoryDoNotUse;
-  final AirbyteGithubStore githubStore;
+  final AirbyteRemoteOssCatalog remoteOssCatalog;
 
-  public WebBackendCheckUpdatesHandler(final ConfigRepository configRepositoryDoNotUse, final AirbyteGithubStore githubStore) {
+  public WebBackendCheckUpdatesHandler(final ConfigRepository configRepositoryDoNotUse, final AirbyteRemoteOssCatalog remoteOssCatalog) {
```

Comment:
Just for my understanding, where/when is this WebBackendCheckUpdatesHandler used?

Reply:
Looks like its used in an api controller https://github.com/airbytehq/airbyte-platform/blob/main/airbyte-server/src/main/java/io/airbyte/server/apis/WebBackendApiController.java#L46